config/main.go

```diff
@@ -91,6 +92,9 @@ var (
 	// FieldNameReplacements represents map of CamelCased field names to under_score field names if needs replacement
 	FieldNameReplacements map[string]string = make(map[string]string)
 
+	// Sort sort fields and messages by name (otherwise, will keep the order as it was in .proto file)
+	Sort bool = false
```

Comment:
nit: bool is false by default
```suggestion
	Sort bool
```